goodtables/i18n/fr/LC_MESSAGES/goodtables.po

```diff
@@ -16,20 +16,24 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.3.4\n"
+"Generated-By: Babel 2.7.0\n"
 
 #: goodtables/inspector.py:71
 msgid "Dataset inspection has reached {} table(s) limit"
 msgstr "L'inspection du jeu de données a atteint la limite de {} table(s)"
 
-#: goodtables/inspector.py:268
+#: goodtables/inspector.py:278
 msgid "Table \"{}\" inspection has reached {} row(s) limit"
 msgstr "L'inspection de la table \"{}\" a atteint la limite de {} ligne(s)"
 
-#: goodtables/inspector.py:275
+#: goodtables/inspector.py:285
 msgid "Table \"{}\" inspection has reached {} error(s) limit"
 msgstr "L'inspection de la table \"{}\" a atteint la limite d'erreur(s) de {}"
 
+#: goodtables/inspector.py:355
+msgid "The data source could not be successfully decoded with utf-8 encoding"
+msgstr "La source des données n'a pas pu être décodée avec l'encodage utf-8"
+
```

Comment:
I would commit only the messages we add not all the comment updates above, just to make merging easier in the future.